### ISSUE TITLE
Do not regenerate AndroidManifest post build

### ISSUE THREAD

**Issue by lacostej** [facebook/facebook-sdk-for-unity#29]
The FacebookPostProcess currently modifies the default `AndroidManifest.xml` [post build](https://github.com/facebook/facebook-sdk-for-unity/blob/master/Facebook.Unity/Assets/FacebookSDK/SDK/Editor/FacebookPostprocess.cs#L62-L66). 
This decision is based on whether the default Unity3d `AndroidManifest.xml` has been modified to contain the expected Facebook configurations.

In our project, we decided to use an AndroidManifest.xml fragment instead. Our default Unity3d `AndroidManifest.xml` should thus not contain the facebook changes and the Postprocess operation causes confusion.

Also given that we build multiple product variants from the same source tree, we use pre-build techniques to customize that fragment AndroidManifest based on our needs. The default generated one isn't necessarily the one we want to have.

We would recommend to not call `GenerateManifest()` at all in the Post build or at least to make it so that we can disable that behaviour without having to patch the SDK at each update. Also the use of an AndroidManifest fragment could be a good thing (e.g.  `Assets/Plugins/Android/Facebook/AndroidManifest.xml`)

**Comment by bblpny** [comment]
I agree and believe this should be optional as well, It seems unnecessary to have this be automated in the first place.
